flutter_apns_only/ios/Classes/FlutterApnsPlugin.swift

```diff
@@ -215,6 +215,8 @@ func getFlutterError(_ error: Error) -> FlutterError {
             let shouldShow = (result as? Bool) ?? false
             if shouldShow {
                 completionHandler([.alert, .sound])
+                let userInfo = FlutterApnsSerialization.remoteMessageUserInfo(toDict: userInfo)
```

Comment:
this code can be moved outside of if else, since it's duplicated

Reply:
done